cmd/context/create.go

```diff
@@ -243,6 +243,15 @@ func (c *ContextCommand) UseContext(ctx context.Context, ctxOptions *ContextOpti
 	return nil
 }
 
+// getClusterMetadata runs the user query GetClusterMetadata and returns the response
+func getClusterMetadata(ctx context.Context, namespace string, okClientProvider oktetoClientProvider) (types.ClusterMetadata, error) {
+	okClient, err := okClientProvider.Provide()
+	if err != nil {
+		oktetoLog.Infof("error providing okteto client: %v", err)
```

Comment:
I think it's better to move logging responsability up, in the calling fn, so that this function is not interacting with logs logic

Reply:
i've reviewed and changed this